src/main/java/com/rocketden/main/dto/room/RoomMapper.java

```diff
@@ -5,6 +5,8 @@
 
 public class RoomMapper {
 
+    protected RoomMapper() {}
+
     private static final ModelMapper mapper = new ModelMapper();
 
     public static CreateRoomResponse entityToCreateResponse(Room entity) {
```

Comment:
Suggestion: add tests for this class (verify additional fields are being mapped correctly)

Reply:
Done!